src/Items/ItemBow.h

```diff
@@ -69,17 +69,12 @@ class cItemBowHandler :
 		}
 
 		// Create the arrow entity:
-		cArrowEntity * Arrow = new cArrowEntity(*a_Player, Force * 2);
-		if (Arrow == nullptr)
+		auto ArrowPtr = cpp14::make_unique<cArrowEntity>(*a_Player, Force * 2);
+		auto Arrow = ArrowPtr.get();
+		if (!Arrow->Initialize(std::move(ArrowPtr), *a_Player->GetWorld()))
```

Comment:
Could someone remind me what ill-effects would happen again? (It's been so long...) I cannot identify a link between the `Arrow` pointer variable, the std::move of `ArrowPtr`, and `this` because I am under the impression that everything is initialised and at no point is anything destroyed.